src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -16,6 +16,31 @@
     'StreamWithdrawal': np.zeros(12),  # Surface Water Withdrawal/Extraction
     'GroundWithdrawal': np.zeros(12),  # Groundwater Withdrawal/Extraction
     'PcntET': np.ones(12),  # Percent monthly adjustment for ET calculation
+    'Landuse': ['Hay/Past',   # Maps NLCD 81
+                'Cropland',   # Maps NLCD 82
+                'Forest',     # Maps NLCD 41, 42, 43, 52
+                'Wetland',    # Maps NLCD 90, 95
+                'Disturbed',       # Does not map to NLCD
+                'Turfgrass',       # Does not map to NLCD
+                'Open_Land',  # Maps NLCD 21, 71
+                'Bare_Rock',  # Maps NLCD 12, 31
+                'Sandy_Areas',     # Does not map to NLCD
+                'Unpaved_Road',    # Does not map to NLCD
+                'Ld_Mixed',   # Maps NLCD 22
+                'Md_Mixed',   # Maps NLCD 23
+                'Hd_Mixed',   # Maps NLCD 24
+                'Ld_Residential',  # Does not map to NLCD
+                'Md_Residential',  # Does not map to NLCD
+                'Hd_Residential',  # Does not map to NLCD
```

Comment:
Land use, and several other fields, are referenced throughout the model code by these enum values https://github.com/WikiWatershed/gwlf-e/blob/develop/gwlfe/enums.py#L64. These are more like glorified strings rather than true enums, due to lack of language support. Still, I would recommend using these values where possible so we can write logic against these symbolic "enums" instead of having to sprinkle string matching logic throughout.